File: src/pocketcloud/cloud/event/impl/server/ServerStopEvent.php
<?php

namespace pocketcloud\cloud\event\impl\server;

use pocketcloud\cloud\server\CloudServer;

class ServerStopEvent extends ServerEvent {

    public function __construct(
        CloudServer $server,
        private readonly bool $force
    ) {
        parent::__construct($server);
    }

    public function isForce(): bool {
        return $this->force;
    }
}

File: src/main/java/de/pocketcloud/cloud/event/impl/server/ServerStopEvent.java
package de.pocketcloud.cloud.event.impl.server;

import de.pocketcloud.cloud.server.CloudServer;
import lombok.Getter;

public final class ServerStopEvent extends ServerEvent {

    @Getter
    private final boolean force;

    public ServerStopEvent(CloudServer server, boolean force) {
        super(server);
        this.force = force;
    }
}
